casper/src/main/scala/coop/rchain/casper/Estimator.scala

```diff
@@ -40,8 +40,11 @@ final class Estimator[F[_]: Sync: Log: Metrics: Span](
     Span[F].trace(Tips0MetricsSource) {
       for {
         latestMessageHashes <- dag.latestMessageHashes
-        _                   <- Span[F].mark("latest-message-hashes")
-        result              <- tips(dag, genesis, latestMessageHashes)
+        _ <- Log[F].info(s"latestMessages: \n ${latestMessageHashes
+              .map(lm => (PrettyPrinter.buildString(lm._1) + " -> " + PrettyPrinter.buildString(lm._2)))
+              .mkString("\n")}")
+        _      <- Span[F].mark("latest-message-hashes")
+        result <- tips(dag, genesis, latestMessageHashes)
```

Comment:
@nzpr Do you want to leave these logging messages?

Reply:
Yes, this is not needed

Reply:
Reverted.